### DISCUSSION TITLE
setAlphaCutoff is not a function

### DISCUSSION THREAD

**Discussion by MeangNy** [google/model-viewer#3543] [Q&A] [answered]
### Description

I can't use function setAlphaCutoff 
![image](https://user-images.githubusercontent.com/46337343/174879272-3980e86f-99c8-4b6c-b0a5-93c587cb0b9d.png)

#### Live Demo

https://cobalt-tan-stage.glitch.me/

### Browser Affected

- [X] Chrome, version: 102.0.5005.115
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

**Comment by elalish** [comment, marked as answer]
that function is on a `Material`: https://modelviewer.dev/examples/scenegraph/#pickMaterialExample, so you'll want something like `mv.model.materials[0].setAlphaCurtoff(0.5);`
